library/consul

```diff
@@ -6,10 +6,10 @@ GitRepo: https://github.com/hashicorp/docker-consul.git
 GitCommit: 20657edd8a3d66ee155b1d050a7a9400e8ab3bdd 
 Directory: 0.X
 
-Tags: 1.6.4, 1.6
+Tags: 1.6.5, 1.6
 Architectures: amd64, arm32v6, arm64v8, i386
 GitRepo: https://github.com/hashicorp/docker-consul.git
-GitCommit: 59bdf0ae8863e3166aadc81e18f8781a52a07136
+GitCommit: 5fdce521f44a1cd56d3f5fb92e5732f2ae516d3e
```

Comment:
In order to pull this commit from a branch other than `master`, we'll need to inform `bashbrew` how/where to fetch it from (GitHub doesn't support fetching arbitrary commits, unfortunately):

```suggestion
GitFetch: refs/heads/consul-1.6.5
GitCommit: 5fdce521f44a1cd56d3f5fb92e5732f2ae516d3e
```